problems/[230710] 35. Search Insert Position.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * https://leetcode.com/problems/search-insert-position/
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number}
+ */
+
+const searchInsert = function (nums, target) {
+  // 접근 방식: O(log n) 으로 탐색하라고 했으니 이진 탐색으로 풀어야겠다.
+  // 시간 복잡도: O(log n)
+  // 공간 복잡도: O(1)
+
+  let left = 0;
+  let right = nums.length - 1;
+  let mid;
+
+  while (left <= right) {
+    mid = Math.floor((left + right) / 2);
```

Comment:
```suggestion
    const mid = Math.floor((left + right) / 2);
```
while문 안에서 지역변수로 설정할 수도 있을 것 같습니다.